inference-engine/tests/functional/plugin/cpu/test_utils/fusing_test_utils.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "fusing_test_utils.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace CPUTestUtils {
+
+
+std::string CpuTestWithFusing::getTestCaseName(fusingSpecificParams params) {
+    std::ostringstream result;
+    std::shared_ptr<ngraph::Function> postFunction;
+    std::vector<postNode> postNodes;
+    std::vector<std::string> fusedOps;
+    std::tie(postFunction, postNodes, fusedOps) = params;
+
+    if (postFunction) {
+        result << "_Fused=" << postFunction->get_friendly_name();
+    } else if (!postNodes.empty()) {
+        result << "_Fused=";
+        const char* separator = "";
+        for (const auto& item : postNodes) {
+            result << separator << item.name;
+            separator = ",";
+        }
+    }
+
+    return result.str();
+}
+
+std::shared_ptr<ngraph::Node>
+CpuTestWithFusing::modifyGraph(const ngraph::element::Type &ngPrc, ngraph::ParameterVector &params, const std::shared_ptr<ngraph::Node> &lastNode) const {
```

Comment:
Totally not clear from name that this method does. We need either change the name or add description in header.

Reply:
Dedicated comments were added 9a2b538.